crates/iota-sdk-ffi/src/transaction_builder/client_builder.rs

```diff
@@ -0,0 +1,383 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    collections::HashMap,
+    sync::{Arc, RwLock, RwLockReadGuard, RwLockWriteGuard},
+    time::Duration,
+};
+
+use iota_sdk::{graphql_client::WaitForTx, types::Input};
+
+use crate::{
+    crypto::simple::SimpleKeypair,
+    error::Result,
+    graphql::GraphQLClient,
+    transaction_builder::ptb_arg::{MoveArg, PTBArgument},
+    types::{
+        address::Address,
+        graphql::DryRunResult,
+        move_package::MovePackageData,
+        object::ObjectId,
+        struct_tag::Identifier,
+        transaction::{Argument, Transaction, TransactionEffects},
+        type_tag::TypeTag,
+    },
+};
+
+/// A builder for creating transactions. Use `finish` to finalize the
+/// transaction data.
```

Comment:
Probably good to mention the client here and that it makes it easier, otherwise from looking at the doc comments  both tx builders are the same